src/views/pages/Login/index.vue

```diff
@@ -55,31 +51,44 @@ export default Vue.extend({
 </script>
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
-<style lang="scss" scoped>
-.login_form {
-  width: 22em;
-  margin: auto;
-  margin-top: 4em;
-  flex-direction: column;
-  display: flex;
-  align-items: flex-end;
-}
+<style lang="scss">
 
-.login_form_feedback {
+#login-page {
   display: flex;
-  width: 22em;
-  justify-content: space-between;
-}
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
 
-.login_button {
-  width: 4em;
-}
+  .heading {
+    align-self: flex-start;
+  }
 
-.error {
-  display: flex;
-  margin: 0;
-  color: red;
-  text-align: left;
-  align-items: center;
+  .login_form {
+    width: 22em;
+    flex-direction: column;
+    display: flex;
+    align-items: flex-end;
+    margin-bottom: 15vh;
+  }
+
+  .login_form_feedback {
+    display: flex;
+    width: 22em;
+    justify-content: space-between;
+  }
+
+  .login_button {
+    width: 4em;
```

Comment:
what was wrong with the default?

Reply:
IIRC I think the button ended up being very wide, taking up the whole space.